newrelic/resource_newrelic_alert_policy_test.go

```diff
@@ -15,7 +15,7 @@ func TestAccNewRelicAlertPolicy_Basic(t *testing.T) {
 	resourceName := "newrelic_alert_policy.foo"
 	rName := acctest.RandString(5)
 
-	resource.ParallelTest(t, resource.TestCase{
+	resource.Test(t, resource.TestCase{
```

Comment:
I _think_ these can use `resource.ParallelTest` again now that we figured out that preliminary oddity with that other test.

Reply:
Ah yes, this was for testing.